leopard.go

```diff
@@ -1,11 +1,8 @@
-//go:build leopard
```

Comment:
If we're removing this, then should the CI also be updated to not build with the `leopard` tag, and not require installing `libleopard`?

Reply:
Done.